tests/platform_tests/cli/test_show_platform.py

```diff
@@ -336,6 +336,47 @@ def test_show_platform_fan(duthosts, enum_supervisor_dut_hostname):
                   " No Fans are displayed with OK status on '{}'".format(duthost.hostname))
 
 
+def check_show_platform_sensor_output(cmd, duthost):
+    """
+    @summary: Run and verify output of `show platform [voltage|current]`. Expected output
+              is "Sensor Not detected" or a table of sensor status data with 8 columns.
+    """
+    num_expected_clos = 8
+
+    logging.info("Verifying output of '{}' on '{}'...".format(cmd, duthost.hostname))
+    raw_output_lines = duthost.command(cmd)["stdout_lines"]
+
+    pytest_assert(len(raw_output_lines) > 0, "There must be at least one line of output on '{}'".format(hostname))
+    if len(raw_output_lines) == 1:
+        pytest_assert(raw_output_lines[0].strip() == "Sensor Not detected",
```

Comment:
Are we sure it's `Not` with a capital `N`? I think it'd better if we can lowercase `raw_output_lines[0].strip()` and then do the comparison, so we don't need to worry about the cases.